src/java.desktop/share/classes/javax/swing/filechooser/FileSystemView.java

```diff
@@ -255,6 +256,54 @@ public Icon getSystemIcon(File f) {
         }
     }
 
+   /**
+    * Icon for a file, directory, or folder as it would be displayed in
+    * a system file browser for the requested size.
+    *
+    * The default implementation gets information from the
+    * <code>ShellFolder</code> class. Whenever possible the icon
```

Comment:
```suggestion
    * <code>ShellFolder</code> class. Whenever possible, the icon
```

Do we continue using `<code>` elements? I thought that `{@code}` is the preferred way to markup class names.